app/src/main/java/org/oppia/android/app/player/state/StateFragmentPresenter.kt

```diff
@@ -451,6 +464,14 @@ class StateFragmentPresenter @Inject constructor(
   /** Returns the checkpoint state for the current exploration. */
   fun getExplorationCheckpointState() = explorationCheckpointState
 
+  fun handleOnSavedInstance(): RawUserAnswer {
+    return if (isConfigChangeStateRetentionEnabled.value) {
+      viewModel.getRawUserAnswer(recyclerViewAssembler::getPendingAnswerHandler)
+    } else {
+      RawUserAnswer.getDefaultInstance()
+    }
```

Comment:
```suggestion
    } else RawUserAnswer.getDefaultInstance()
```

Small change for conciseness.

Reply:
Done